tests.py

```diff
@@ -115,7 +94,7 @@ def test_run_no_such_program(self):
         should_start_with = "could not run ['there-is-really-no-such-program']:"
         self.assertTrue(
             str(cm.exception).startswith(should_start_with),
-            '\n%r does not start with\n%r' % (str(cm.exception),
+            '\n{!r} does not start with\n{!r}'.format(str(cm.exception),
                                               should_start_with))
```

Comment:
This line is now misindented: it was matching the position of the `(` in the previous line, and no longer does. (The pain of these indentation adjustments is one of the reasons I've lately switched to a more Black-inspired indentation style with a line break right after the opening `(` and a 4-space increase of indentation on the next line.)

If we used an f-string here, maybe it would fit without wrapping?